### PR TITLE
Email form

### PR DESCRIPTION
**Pull request by kktai1512**
Use formspree

### CODE REVIEW

**Comment by kktai1512** [comment]
[contacts page](https://trello.com/c/5L55A75o/20-contacts-page)

**Comment by kktai1512** [comment]
[Create contacts page FORM (with form to email service)](https://trello.com/c/0q0negd6/29-create-contacts-page-form-with-form-to-email-service)

File: src/components/Button.tsx [hunk 1]
```diff
@@ -6,18 +6,27 @@ type Props = {
...
+    if (props.onClick) {
+      return props.onClick && props.onClick()
+    }
+    return window.open(props.link, '_blank')
+  }
   return (
     <BaseButton fontSize={props.fontSize} isActive={props.isActive}>
       {/* eslint-disable-next-line security/detect-non-literal-fs-filename */}
-      <AntdButton type="text" onClick={() => window.open(props.link, '_blank')}>
+      <AntdButton type="text" onClick={() => onClick()}>
```
Comment (marcus-ong-qy) [inline src/components/Button.tsx:29]: same, `onClick={onClick}` should suffice

**Review by marcus-ong-qy — CHANGES_REQUESTED** [review]
Other than that the error messages are not showing on form submit, the form itself lgtm!

**Review by marcus-ong-qy — APPROVED** [review]

**Comment by marcus-ong-qy** [comment]
lgtm! do consider reviewing my comments and merge soon!

**Pull request merged**